packages/devtools_app/lib/src/inspector/inspector_tree_controller.dart

```diff
@@ -542,6 +585,151 @@ class InspectorTreeController extends Object {
       }
     }
   }
+
+  /* Search support */
+  @override
+  void previousMatch() {
+    super.previousMatch();
+    final currentSearchMatches = searchMatches.value;
+    for (int i = 0; i < currentSearchMatches.length; i++) {
+      final searchMatch = currentSearchMatches[i];
+
+      if (searchMatch.isSelected) {
+        final rowToSelect = i == 0
+            ? currentSearchMatches[currentSearchMatches.length - 1]
+            : currentSearchMatches[i - 1];
+
+        onSelectRow(rowToSelect);
+        break;
+      }
+    }
+  }
+
+  @override
+  void nextMatch() {
+    super.nextMatch();
+    final currentSearchMatches = searchMatches.value;
+    for (int i = 0; i < currentSearchMatches.length; i++) {
+      final searchMatch = currentSearchMatches[i];
+
+      if (searchMatch.isSelected) {
+        final rowToSelect = i == currentSearchMatches.length - 1
+            ? currentSearchMatches[0]
+            : currentSearchMatches[i + 1];
+
+        onSelectRow(rowToSelect);
+        break;
+      }
+    }
+  }
+
+  void dispose() {
+    if (_searchDebounce?.isActive ?? false) {
+      _searchDebounce.cancel();
+    }
+  }
+
+  Future _searchOperation;
+  Timer _searchDebounce;
+
+  @override
+  List<InspectorTreeRow> matchesForSearch(String search) {
```

Comment:
Can you add support for searching through previous matches? See the super class for method signature (you may need to sync to head - this framework was added a couple of weeks ago)

Reply:
Sure, this is something I'll look into early next week